spring-cloud-task-core/src/test/java/org/springframework/cloud/task/configuration/TaskPropertiesTests.java

```diff
@@ -0,0 +1,55 @@
+package org.springframework.cloud.task.configuration;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+import org.junit.runners.Suite.SuiteClasses;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.cloud.task.repository.TaskRepository;
+import org.springframework.cloud.task.repository.support.SimpleTaskRepository;
+import org.springframework.cloud.task.repository.support.TaskExecutionDaoFactoryBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.junit4.SpringRunner;
+
+
+@RunWith(Suite.class)
+@SuiteClasses({
+		TaskPropertiesTests.CloseContextEnableTest.class,
+		TaskPropertiesTests.CloseContextEnabledTest.class,
+		
+})
+
+@DirtiesContext
+public class TaskPropertiesTests {	
+	@Test
+	public void test() {
```

Comment:
A tad picky but we can also check the property itself.   For example:
```
	@Autowired
	TaskProperties taskProperties;
	
	@Test
	public void test() {
		assertThat(taskProperties.getClosecontextEnabled(), is(false));
	}
```

I can add during merge if you have no objections.